lib/AlexaSkill.js

```diff
@@ -131,6 +131,10 @@ class AlexaSkill {
           // call all onRequestStarted callbacks serially.
           return Promise.mapSeries(this.getOnRequestStartedHandlers(), fn => fn(alexaEvent, reply))
             .then(() => {
+              if (alexaEvent.request.type === 'SessionEndedRequest' && _.get(alexaEvent, 'request.reason') === 'ERROR') {
+                throw new Error(_.get(alexaEvent, 'request.error'));
```

Comment:
perhaps we should have some custom error so we can check for error type in the error handler? 
https://github.com/mediarain/voxa/blob/master/lib/Errors.js